daprdocs/content/en/python-sdk-docs/_index.md

```diff
@@ -9,46 +9,109 @@ no_list: true
-    - [**FastAPI extension**]({{< ref python-fastapi.md >}}) for adding Dapr actor capabilities to FastAPI applications
-    - [**Flask extension**]({{< ref python-flask.md >}}) for adding Dapr actor capabilities to Flask applications
+## Installation
 
-## Install the Dapr module
+To get started with the Python SDK, install the Dapr Python SDK package
 
 {{< tabs Stable Development>}}
 
 {{% codetab %}}
+<!--stable-->
 ```bash
 pip install dapr
 ```
 {{% /codetab %}}
 
 {{% codetab %}}
-{{% alert title="Note" color="warning" %}}
-The development package will contain features and behavior that will be compatible with the pre-release version of the Dapr runtime. Make sure to uninstall any stable versions of the Python SDK before installing the dapr-dev package.
+<!--dev-->
+> **Note:** The development package will contain features and behavior that will be compatible with the pre-release version of the Dapr runtime. Make sure to uninstall any stable versions of the Python SDK before installing the dapr-dev package.
 {{% /alert %}}
 
 ```bash
 pip install dapr-dev
 ```
+
 {{% /codetab %}}
 
 {{< /tabs >}}
 
+
 ## Try it out
 
-Clone the Python SDK repo to try out some of the [examples](https://github.com/dapr/python-sdk/tree/master/examples).
+Clone the Python SDK repo.
 
 ```bash
 git clone https://github.com/dapr/python-sdk.git
 ```
 
+Walk through the Python quickstarts, tutorials, and examples to see Dapr in action:
+
+| SDK samples | Description |
+| ----------- | ----------- |
+| [Quickstarts]({{< ref quickstarts >}}) | Experience Dapr's API building blocks in just a few minutes using the Python SDK. |
+| [SDK samples](https://github.com/dapr/python-sdk/tree/master/examples) | Clone the SDK repo to try out some examples and get started. |
+| [Bindings tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/bindings) | See how Dapr Python SDK works alongside other Dapr SDKs to enable bindings. |
+| [Distributed Calculator tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/distributed-calculator/python) | Use the Dapr Python SDK to handle method invocation and state persistent capabilities. |
+| [Hello World tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/hello-world) | Learn how to get Dapr up and running locally on your machine with the Python SDK. |
+| [Hello Kubernetes tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/hello-kubernetes) | Get up and running with the Dapr Python SDK in a Kubernetes cluster. |
+| [Observability tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/observability) | Explore Dapr's metric collection, tracing, logging and health check capabilities using the Python SDK. |
+| [Pub/sub tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/pub-sub) | See how Dapr Python SDK works alongside other Dapr SDKs to enable pub/sub applications. |
+
+
+## Available packages
```

Comment:
This is incorrect.

In Python the Actor SDK is part of the main SDK, it's not a separate SDK. Please update these cards to match the information you removed above.

We have the main SDK (which includes Actors - no need to call that out specifically), we have the following Extension SDKs: Workflows (yes this is an extension SDK - the package is named `dapr-ext-workflow`), Flask, FastAPI and gRPC. Those latter 3 are primarily for handling PubSub subscriptions, Input Bindings etc in your app more easily.

Reply:
@berndverst working on this - another Q: should the `python-workflow.md` doc be moved in the TOC to sit under "Extensions"?

Reply:
Technically yes - but that's only because today we have the main SDK and extensions for receiving requests from the sidecar (which arguably the Workflow authoring SDK does too). We can probably leave it in place where it is to make it more easily discoverable however.

By the way, can you update the docs to say that the Workflow SDK is Alpha? Once we version it as 1.0 we will consider it stable (and hence not make potential breaking changes)